libtsuba/src/RDGCore.cpp

```diff
@@ -122,20 +122,17 @@ EnsureTypeLoaded(const katana::Uri& rdg_dir, tsuba::PropStorageInfo* psi) {
   return katana::ResultSuccess();
 }
 
-katana::Result<tsuba::PropStorageInfo>
+katana::Result<tsuba::PropStorageInfo*>
```

Comment:
If we're going to return a pointer anyway, maybe take advantage of `nullptr` and make that the return value for no property found?

Reply:
Yeah, that is probably fine. Do we not usually return Results of pointers?

Reply:
For something like this where not found isn't necessarily an error I think it makes sense. I mean, we've already gotten the error code check wrong once 😅

Reply:
I'd like to live in a world where NO error code is ever interpreted at a higher level as a non-error.  That only happens in a very few cases and I don't like it.